docs/releases/1.9.4.txt

```diff
@@ -4,9 +4,10 @@ Django 1.9.4 release notes
 
 *Under development*
 
-Django 1.9.4 fixes several bugs in 1.9.3.
+Django 1.9.4 fixes a regression in the 1.9.3 security release.
```

Comment:
If you feel this should be released asap, then let's just say "a regression on Python 2 in the 1.9.3 security release where utils.http.is_safe_url() crashes on bytestring URLs (`:ticket:`...)" and remove the "Bugfixes" section.